Tiltfile

```
@@ -166,18 +178,31 @@ def deploy_cspo():
 
     # Set up an image build for the provider. The live update configuration syncs the output from the local_resource
     # build into the container.
-    docker_build_with_restart(
-        ref = "ghcr.io/sovereigncloudstack/cspo-staging",
-        context = "./.tiltbuild/",
-        dockerfile_contents = tilt_dockerfile_header_cspo,
-        target = "tilt",
-        entrypoint = entrypoint,
-        only = "manager",
-        live_update = [
-            sync(".tiltbuild/manager", "/manager"),
-        ],
-        ignore = ["templates"],
-    )
+    if settings.get("local_mode"):
+        docker_build_with_restart(
+            ref = "ghcr.io/sovereigncloudstack/cspo-staging",
+            context = ".",
+            dockerfile_contents = tilt_dockerfile_header_cspo,
+            target = "tilt",
+            entrypoint = entrypoint,
+            live_update = [
+                sync(".tiltbuild/manager", "/manager"),
+                sync(".release", "/tmp/cluster-stacks"),
```

Shouldn't it be this?
```suggestion
                sync(".release", "/tmp/downloads/cluster-stacks"),
```